packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -1659,9 +1662,25 @@ export default class NewVm extends BaseComponent {
       showAdvanced,
       tags,
     } = this.state.state
-    const { isAdmin } = this.props
+    const { isAdmin, template } = this.props
     const { formatMessage } = this.props.intl
     const isHvm = this._isHvm()
+    const hasBiosStings =
+      template !== undefined && !isEmpty(template.bios_strings)
+    const copyHostBiosStrings_ =
+      isAdmin && isHvm ? (
+        <label>
+          <input
+            checked={copyHostBiosStrings}
+            className='form-control'
+            disabled={hvmBootFirmware === 'uefi' || hasBiosStings}
```

Comment:
Should the checkbox be checked when `hasBiosStrings` is true? If the template contains the BIOS strings, the VM will too, right?